sirepo/pkcli/admin.py

```diff
@@ -104,9 +104,9 @@ def setup_dev_proprietary_code(sim_type, rpm_url):
     assert pkconfig.channel_in('dev'), \
         'Only to be used in dev. channel={}'.format(pkconfig.cfg.channel)
 
+    s = sim_data.get_class(sim_type)
```

Comment:
remove this change, because it conflicts the #2586

Reply:
https://github.com/radiasoft/sirepo/pull/2580/commits/d605d20b82b846260157ed0ad2bf276556bc4b0a